pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/FormComponent.java

```diff
@@ -19,12 +21,6 @@
 
     protected transient boolean hideDisplayNumber;
 
-    @NotNull
-    @SerializedName("parameters")
-    // derived classes should fill this map explicitly.  this is where the client
-    // gets the list of parameters
-    protected Map<String, Object> parameters = new LinkedHashMap<>();
```

Comment:
From now on each child class is responsible for holding its serializable fields (instead of stuffing a parent's `Map`).